src/components/layout.js

```diff
@@ -45,6 +46,7 @@ const _Layout = ({ children }) => {
         />
       </Helmet>
       <Header siteTitle={data.site.siteMetadata.title} />
+      <Hero />
```

Comment:
Hero doesnt go in the layout. We don't necessarily want it on every page. We want to include it at the page/route level.

Reply:
Moved it to index only. I had a feeling it was in the wrong spot.